tests/cli/cdk_smoke_tests/cdk-basic.bats

```diff
@@ -157,3 +160,7 @@ setup_file() {
     assert_output --partial "Connector runs with process id"
 }
 
+@test "List connectors from hub" {
+    run timeout 15s $CDK_BIN hub list
```

Comment:
this test needs access to Hub, but accessing `fluvio cloud` in CI is a known issue for now (see #3559). We're working on it.